app/src/main/java/amazingme/activities/app/UserProfileActivity.java

```diff
@@ -40,11 +40,11 @@ public void onClick(View view) {
 
             @Override
             public void onClick(View v) {
-                Parent parent = getContext().getUserContext().getParent();
+                Parent parent = getAppContext().getUserContext().getParent();
```

Comment:
can we hide this in the superclass to be just getUserContext()? 

i thought we said that was a good middle ground but remind me if i'm wrong.